ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
If you're going to add similar functionality later then go ahead with a -0 from me. But if you're going to add stuff related to this later then why not remove them in the upcoming changes rather than in this pr with otherwise unrelated changes?

Also this pattern of "entering"/"leaving" messages is used in other places. I'd prefer changing all of them or none, e.g.:

https://github.com/ament/ament_package/blob/master/ament_package/template/prefix_level/local_setup.bat.in#L3